integration/kubernetes/helm/alluxio/templates/alluxio-ha-master.yaml

```diff
@@ -0,0 +1,153 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+{{- if eq .Values.journal.type "EMBEDDED" }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: alluxio-pv-claim
+spec:
+  storageClassName: {{ .Values.volumes.master.journal.storageClass }}
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: {{ .Values.volumes.master.journal.size | quote }}
+{{- end}}
+
+{{- $root := . -}}
```

Comment:
This is due to Go template's limitation discussed in 
[Variables not accessed correctly in loop](https://github.com/helm/helm/issues/1055)